locale/fi/LC_MESSAGES/django.po

```diff
@@ -1427,6 +1427,10 @@ msgstr ""
 "<p>Haluatko varmasti estää tämän käyttäjän pääsyn kurssille?</p><p>Estetyt "
 "käyttäjät eivät pysty ilmoittautumaan kurssille uudelleen.</p>"
 
+#: course/templatetags/base.py
+msgid "GRADER_PROBLEMS_ALERT"
+msgstr "Automaattisessa arvioinnissa on tällä hetkellä ongelmia. Palautteen saamisessa saattaa esiintyä viivettä."
```

Comment:
This message and the English version seem too long, which means that the next time someone runs ``makemessages`` it's going to shorten these lines automatically. It happened to me too...